src/libs/data-utils.js

```diff
@@ -445,6 +445,21 @@ export const EntityRenamer = ({ entityType, entityName, workspaceId: { namespace
   ])
 }
 
+export const createEntitySet = ({ entities, rootEntityType, newSetName, workspaceId: { namespace, name } }) => {
```

Comment:
we pulled this code out of CreateSetAndLaunch (in LaunchAnalysisModal) so we could call it from our new function